intake/catalog/tests/test_catalog_save.py

```diff
@@ -10,6 +10,8 @@
 def test_catalog_description():
```

Comment:
Was this test failing? It'd be good to make a test that would have caught the missing key in `__init__`.

Reply:
It wasn't failing, but was leaving the created test catalog in a non-temporary directory.